charts/datree-admission-webhook/templates/cluster-scan-cronjob.yaml

```diff
@@ -52,9 +52,10 @@ metadata:
   annotations: {{ toYaml . | nindent 4 }}
   {{- end }}
 spec:
+  concurrencyPolicy: Forbid
```

Comment:
I believe this is redundant

Reply:
I added this during the testing (out of fear rather than sense), removed it now.